spec/controller_services/shopping_list_items_controller/create_service_spec.rb

```diff
@@ -46,6 +46,14 @@
           expect(aggregate_list).to have_received(:add_item_from_child_list).with(shopping_list.list_items.last)
         end
 
+        it 'updates the list model itself' do
+          t = Time.now + 3.days
+          Timecop.freeze(t) do
+            perform
+            expect(shopping_list.reload.updated_at).to be_within(0.005.seconds).of(t)
```

Comment:
Although Timecop is exact, differences with timestamps in CI, where the last four digits are truncated to 0, make it necessary to use `be_within` here instead of `eq`.